packages/deployer/.mocharc.json

```diff
@@ -1,5 +1,6 @@
 {
   "spec": ["test/unit/**/*.test.js", "test/integration/**/*.test.js"],
+  "timeout": 10000,
```

Comment:
This is necessary because *sometimes* the github actions CI is taking a little longer than the default `2secs`.